### PR TITLE
Doc: Add GitHub issue templates

### PR DESCRIPTION
**Pull request by emranbm**
In order to clarify the procedures of how to contribute to maturity model, some issue templates are added.

### CODE REVIEW

**Review by littleali — COMMENTED** [review]
I think the better way is using [YAML format](https://docs.github.com/en/communities/using-templates-to-encourage-useful-issues-and-pull-requests/configuring-issue-templates-for-your-repository#creating-issue-forms) to configure our issue templates. Why you didn't use  that?

**Comment by emranbm** [comment]
> I think the better way is using [YAML format](https://docs.github.com/en/communities/using-templates-to-encourage-useful-issues-and-pull-requests/configuring-issue-templates-for-your-repository#creating-issue-forms) to configure our issue templates. Why you didn't use that?

Good idea! I switched to YML format. Please review again.

File: .github/ISSUE_TEMPLATE/edit-item.yml [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+  - type: textarea
+    attributes:
+      label: Description
+      placeholder: Leave blank if this field need no changes.
+    validations:
+      required: false
+  - type: dropdown
+    attributes:
+      label: Maturity Level
+      description: The level you think best fits for this item. (Leave blank if this field need no changes.)
```
Comment (littleali) [inline .github/ISSUE_TEMPLATE/edit-item.yml:31]:
```suggestion
      description: The level you think best fits for this item. (Leave it blank if this field needs no change.)
```

File: .github/ISSUE_TEMPLATE/feedback.yml [hunk 2]
```diff
@@ -0,0 +1,38 @@
+name: Feedback
+description: Share how your experience was using QMM, or what is preventing to use it as a software quality maturity model.
```
Comment (littleali) [inline .github/ISSUE_TEMPLATE/feedback.yml:2]:
```suggestion
description: Share how was your experience using QMM, or what is preventing you to use it as a software quality maturity model.
```

File: .github/ISSUE_TEMPLATE/feedback.yml [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+    attributes:
+      label: How did you find QMM?
+      multiple: false
+      options:
+        - Friends / Coworkers
+        - Social Media / Internet
+        - Other
+  - type: dropdown
+    attributes:
+      label: Are you using QMM for your product(s), currently?
```
Comment (littleali) [inline .github/ISSUE_TEMPLATE/feedback.yml:20]:
```suggestion
      label: Are you currently, using QMM for your product(s)?
```

File: .github/ISSUE_TEMPLATE/feedback.yml [hunk 4]
```diff
@@ -0,0 +1,38 @@
...
+      multiple: false
+      options:
+        - "Yes"
+        - "No"
+  - type: dropdown
+    attributes:
+      label: How do you feel about QMM?
```
Comment (littleali) [inline .github/ISSUE_TEMPLATE/feedback.yml:27]:
```suggestion
      label: QMM is ...
```

File: .github/ISSUE_TEMPLATE/help.yml [hunk 5]
```diff
@@ -0,0 +1,19 @@
+name: Get help
+description: Get help about QMM, its appliances, items' details, etc.
```
Comment (littleali) [inline .github/ISSUE_TEMPLATE/help.yml:2]:
```suggestion
description: Get help about QMM, its application(s), items' details, etc.
```

File: .github/ISSUE_TEMPLATE/new-item.yml [hunk 6]
```diff
@@ -0,0 +1,59 @@
+name: Suggest new item
+description: Suggest a new maturity item.
+labels:
+  - new maturity item
+body:
+  - type: checkboxes
+    attributes:
+      label: Precheck
+      description: Check that current maturity items can't be modified/generalized to include your concern.
```
Comment (littleali) [inline .github/ISSUE_TEMPLATE/new-item.yml:9]:
```suggestion
      description: Are you sure current maturity items can't be modified/generalized to include your concern?
```

File: .github/ISSUE_TEMPLATE/new-item.yml [hunk 7]
```diff
@@ -0,0 +1,59 @@
...
+      options:
+        - label: This item can not be defined below any of the current maturity items.
```
Comment (littleali) [inline .github/ISSUE_TEMPLATE/new-item.yml:11]:
```suggestion
        - label: Yes.
```

File: .github/ISSUE_TEMPLATE/new-item.yml [hunk 8]
```diff
@@ -0,0 +1,59 @@
...
+          required: true
+  - type: markdown
+    attributes:
+      value: |
+        Thanks for introducing a new maturity item. Before filling the next fields, please review maturity items' [style rules](https://maturity.qualitic.ir/catalog/template).
```
Comment (littleali) [inline .github/ISSUE_TEMPLATE/new-item.yml:16]:
```suggestion
        Thanks for proposing a new maturity item. Before  going any further, please review maturity items' [style rules](https://maturity.qualitic.ir/catalog/template).
```

File: .github/ISSUE_TEMPLATE/new-item.yml (hunk 7 above)
Comment (emranbm) [reply .github/ISSUE_TEMPLATE/new-item.yml:11]: It's gonna look like this:
Screenshot from 2022-02-20 12-58-25

So I think the usage of `description` field is just to give a hint and the check box `lable` should be descriptive enough.

**Review by littleali — APPROVED** [review]

File: .github/ISSUE_TEMPLATE/feedback.yml (hunk 3 above)
Comment (emranbm) [reply .github/ISSUE_TEMPLATE/feedback.yml:20]: Not sure it's better :/

**Review by littleali — APPROVED** [review]

**Pull request merged**